Game 1 FlappyBird/FlappyBird/Source/FlappyBird/Private/Character/FlappyCharacter.cpp

```diff
@@ -0,0 +1,53 @@
+﻿// 
+
+
+#include "FlappyBird/Public/Character/FlappyCharacter.h"
+
+#include "EnhancedInputComponent.h"
+#include "EnhancedInputSubsystems.h"
+#include "Camera/CameraActor.h"
+#include "Components/CapsuleComponent.h"
+#include "GameFramework/CharacterMovementComponent.h"
+
+AFlappyCharacter::AFlappyCharacter()
+{
+	PrimaryActorTick.bCanEverTick = true;
+
+	GetCapsuleComponent()->SetSimulatePhysics(true);
```

Comment:
Why not use the checkboxes in engine for this?

Reply:
This instantiates the class itself with the values I want.  So if I were to make multiple blueprints based on this class they would already have the values set. For this scenario I'm only going to have the one flappy bird so I could just do it in engine. But imagine if I wanted to make multiple playable characters.